specs/0106-concordance-search.md

```diff
@@ -12,16 +12,21 @@ Having the ability to search for existing translations is a crucial tool for ass
 
 That being said, Translation Memory is only queried for strings as a whole, even if you "Search in Machinery". If you're interested in the translation of a word "file", but your corpus only uses that word as part of longer strings like "Select a file..." or "Source file uploaded", Machinery won't find any matches.
 
-Alternativelly, you can switch to the All Projects view and use the string list search, which searches for substrings within existing translations for each word separatelly, but that also comes with a few drawbacks. You either lose translation progress in the editor and the state of the string list, or you need to use a separate tab. You also won't be able to search for translations in the projects that aren't active anymore.
+Alternatively, you can switch to the All Projects view and use the string list search, which searches for substrings within existing translations for each word separately, but that also comes with a few drawbacks. You either lose translation progress in the editor and the state of the string list, or you need to use a separate tab. You also won't be able to search for translations in the projects that aren't active anymore.
 
 # Feature explanation
 
-This feature changes the way "Search in Machinery" searches Translation Memory. Instead of searching for fuzzy matches of whole strings stored in Translation Memory, `source` and `target` fields of the `TranslationMemoryEntry` model are searched for each keyword and each phrase within quotes separately.
+This feature changes the way "Search in Machinery" searches Translation Memory. Instead of searching for fuzzy matches of source strings stored in TM, it searches source strings and translations stored in TM the same way as source strings and translations are searched in the string list. Every search keyword is searched separately, unless it's part of a phrase within double quotes - in this case the entire phrase within the quotes is searched as a whole.
```

Comment:
Possibly more readable for the second sentence:

> Instead of searching for fuzzy matches of source strings stored in TM, it searches source strings and translations stored in TM with the same approach used in the string list. Every search keyword…